src/guide/get-started/operate-iroha-via-cli.md

```diff
@@ -0,0 +1,359 @@
+# Operate Iroha via CLI
+
+In this tutorial, we will set up and configure the Iroha client CLI, and perform some basic actions on the Iroha blockchain with it.
+
+You can learn how to:
+  - register [domains](#_3-register-a-domain) and [accounts](#_4-register-an-account)
+  - [mint](#_6-register-and-mint-assets), [transfer](#_7-transfer-assets) and [burn](#_8-burn-assets) assets
```

Comment:
register asset definitions too

Reply:
fixed in commit fec265d9a98b1d356133521d2320837db466d62f